src/libstore/local-store.md

```diff
@@ -0,0 +1,40 @@
+R"(
+
+**Store URL format**: `local`, *root*
```

Comment:
I had to read multiple examples to understand that this notation means an enumeration of store formats. There probably is a better way to express this in a way that is immediately clear, but I don't see one yet.